accounts/modules/role_policies/ci/policies.tf

```diff
@@ -17,6 +17,7 @@ data "aws_iam_policy_document" "ci_permissions" {
       "ecr-public:BatchCheckLayerAvailability",
       "ecr-public:Describe*",
       "ecr-public:Get*",
+      "ecr-public:GetAuthorizationToken",
```

Comment:
In order to get terraform to no-op I needed to add this permission - i'm not certain if it's been added manually or what the consequence will be if I remove it (so it gets added incidentally here).

Reply:
It's from https://github.com/wellcomecollection/platform-infrastructure/pull/213 – I need it to get publishing to ECR Public working in CI.